src/pages/kiman/Main/MainFeed/Othersdata/PushData.js

```diff
@@ -0,0 +1,25 @@
+const PushData = [
```

Comment:
상수 데이터 네이밍 컨벤션 -> UPPER_SNAKE_CASE
```suggestion
const PUSH_DATA = [
```

Reply:
그리고 push data라는 명칭이 어떤 데이터를 담고 있는지 명확하지 않음